go.sum

```diff
@@ -314,6 +316,10 @@ github.com/rogpeppe/go-internal v1.5.2/go.mod h1:xXDCJY+GAPziupqXw64V24skbSoqbTE
 github.com/russross/blackfriday/v2 v2.0.1/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
 github.com/ryancurrah/gomodguard v1.1.0 h1:DWbye9KyMgytn8uYpuHkwf0RHqAYO6Ay/D0TbCpPtVU=
 github.com/ryancurrah/gomodguard v1.1.0/go.mod h1:4O8tr7hBODaGE6VIhfJDHcwzh5GUccKSJBU0UMXJFVM=
+github.com/ryanrolds/sqlclosecheck v0.2.0 h1:lzXPc4X12yhUqHHU2PWl45YGbV4jpCmhcrvgLxlZDeg=
```

Comment:
not sure why this entry is here as the version in go.mod is v0.3.0-rc. Can you help to run _go mod tidy_ once ?

Reply:
I will fix that.

Reply:
I update the version of `sqlclosecheck` while I was working on the PR and didn't run tidy.